### DISCUSSION TITLE
Análisis de requisitos y requerimientos practica 1.

### DISCUSSION THREAD

**Discussion by FaustoPaFl** [Launch-X-Latam/MisionFrontEnd#117] [Q&A]
Hola, alguien me da una idea de que hacer en ésta parte del formulario, siendo más especifico, no entiendo que poner en la parte de modelamiento de negocio. 

"(Especifique un diagrama de negocio que permita entender con claridad que parte del negocio se incluye o se modifica con la solución del requerimiento.
Se sugiere para esta tarea utilice diagramas BPM, diagramas de actividades, o diagramas ad hoc (boceto).)"

Sé a lo que se refiere el texto pero no se concretar la idea... o sea no me entiendo ni a mi... 
Alguna idea?

**Comment by maxkario** [comment]
Tengo esas dudas también. Adicional a eso en que formato se debe entregar la practica. Evidentemente los wireframe si se como entregarlos pero lo demás aun no lo se.

**Comment by samujm** [comment]
Hola! Por lo que yo entiendo es el flujo que va llevando el sistema, como el del ejemplo de la imagen que les puse

![image](https://user-images.githubusercontent.com/69430694/193437284-a1c7bc60-e53a-4803-8d62-bcaf880870fb.png)

pueden leer un poco de diagramas de actividades, les adjunto un video para que se den una idea
https://www.youtube.com/watch?v=YcE9CUqtCw8&ab_channel=Ingenioteka

**Reply by Amhedriel** [reply]
Buenas **samujm**, me parece interesante tu diagrama, creo que el *"Client logs in to the system"* debería estar antes que *"Client´s settings are displayed"* porque al ser satisfactorio el inicio de sesión en el servidor recién ahí  *"Client´s settings are displayed"*.

Hasta donde tengo entendido al colocar tus datos de inicio de sesión se lo envía a los servidores, si lo rechazan te deniegan el acceso y te piden repetir el proceso, si es satisfactorio el servidor inicia tu sesión en el sistema y después te enseña las opciones en pantalla.

Si me equivoco espero me corrijan ahora antes de cometer errores fatales en el futuro.

Saludos y gracias.

**Reply by samujm** [reply]
Es solo un ejemplo que encontré en internet para explicarles el diagrama de negocios, pero gracias

**Comment by FaustoPaFl** [comment]
Gracias, @samujm me ayudaste bastante con esas ideas =)

**Reply by samujm** [reply]
De nada, cualquier duda aqui seguimos :) buen dia!
